src/interceptors/add_idp_token_to_params.js

```diff
@@ -0,0 +1,16 @@
+/**
+   Read `idpToken` from `ctx` and add it to `params`
+
+   Changes to `ctx`:
+
+   - add `params.idpToken`
+ */
+export default class AddIdpTokenToParams {
+  enter({ params, ...ctx }) {
+    const { idpToken } = params;
+    return {
+      ...ctx,
+      params: { ...params, idp_token: idpToken },
+    };
```

Comment:
Is there a reason why `idpClientId` is kept in the context in the `add_idp_client_id_to_params.js` interceptor but `idpToken` is left out here? Not sure if we should only keep all params in the context or just the ones we might need in a forthcoming interceptor.

Reply:
With `idpClientId` I followed the example from `AddClientIdToParams` where the `clientId` is kept in the context and didn't think about it that much. But it seems that we don't need the `idpClientId` later so maybe there is no need to keep it and it would be better to align it with `idpToken`. Or do you think it would make more sense to keep both `idpClientId` and `idpToken`?

Reply:
I'm thinking we could keep everything in the context, so not remove anything. On general level, would it be odd if some interceptors removed data from ctx? Or it could make further development hard if a ctx attribute could be dropped at some point. I'm open for counter arguments but I'd vote not removing anything from the ctx.